src/coap_blockwise.c

```diff
@@ -401,9 +395,9 @@ enum golioth_status golioth_blockwise_get(struct golioth_client *client,
                                           void *callback_arg)
 {
     enum golioth_status status = GOLIOTH_ERR_FAIL;
-    if (!client || !path || !cb)
+    if (!client || !path || !path_prefix || !cb)
```

Comment:
In the other CoAP functions (set, get, delete, etc.), we allow `path_prefix` to be `NULL`

Reply:
Okay, I'll take `path_previx` out of the list (but leave the change for GOLIOTH_ERR_NULL to be returned for other null pointers).